src/oracle/query_builder/subselect.rs

```diff
@@ -0,0 +1,61 @@
+use std::marker::PhantomData;
+
+use diesel::expression::array_comparison::MaybeEmpty;
+use diesel::expression::*;
+use diesel::query_builder::*;
+use diesel::result::QueryResult;
+
+use super::Oracle;
+
+#[derive(Debug, Copy, Clone, QueryId)]
+pub struct Subselect<T, ST> {
```

Comment:
Why does we need that?

Reply:
We could define our own expression (`exists`) but then would probably need to implement everything here instead of using diesel (this yields some compiler errors about unfulfilled traits) (c.f. https://github.com/GiGainfosystems/diesel-oci/blob/feature/more_gst_compat/src/oracle/query_builder/exists.rs#L48)